crate.lua

```diff
@@ -30,13 +31,16 @@ minetest.register_node("castle_storage:crate", {
 		return inv:is_empty("main")
 	end,
 	on_metadata_inventory_move = function(pos, from_list, from_index, to_list, to_index, count, player)
-		minetest.log("action", S("@1 moves stuff in crate at @2", player:get_player_name(), minetest.pos_to_string(pos)))
+		minetest.log("action", S("@1 moves stuff in crate at @2",
```

Comment:
`S()` translations are only decoded by clients, hence this will cause strange output to the server logs. I wonder why nobody has noticed this yet.